50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,334 @@
+  \item Detailed operations guide for handling and operating the satellite
+  \item Where possible, instruction guides for assembly of key components and subsystems that future teams could leverage.
+  \item Test procedures and results for key areas including radio performance, thermal and structural tolerance, and software qualifications
+\end{itemize}
+The published designs also include an arduino-based radio receiver station. The team could develop this component as well, or preferably, leverage other teams and their projects to satisfy this requirement.
+Finally, the team will need to generate a presentation poster that could be used at ImagineRIT or other public SPEX events to highlight the project.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+If this project is conducted over a semester timeline, the team will need to expediently cover the required areas defined below.
+
+This project will take place over three technical domains: electrical, mechanical, and software.
+
+For the electrical systems, the team will need to review the existing PCB designs, primarily to determine whether it is feasible to manufacture and populate them on campus.
+Also the team will need to determine if all components required are still available, and if not, find suitable replacements and make the necessary design changes.
+PCB assembly will be the system with the longest lead time and should be prioritized first to ensure no bottlenecks on the critical path.
+Once assembled, each will need to be tested, and integrated within the system structure.
+
+For the mechanical systems, the team should evaluate the feasibility of manufacturing the frame on-campus.
+Any tweaks to the internal structure and placement of internal components should be locked down as early as possible.
+The mechanical team will also be responsible for testing the frame and whole assembly for verification against thermal and vibration requirements.
+
+For the onboard software, the team should review the existing code, and try to demonstrate functionality on similar hardware to prevent dependency on the electrical team.
+Once the primary electronics are ready, the software should be tested for functionality on the system hardware.
+Table 1 shows a preliminary timeline for the project.
+\autoref{tab:timeline}
+\begin{table*}
+  \caption{Estimated Timeline}
+  \small\centering
+  \begin{tabular}{@{}lll@{}} \toprule
+    Phase & Task & Duration \\ \midrule
+    1 & Review the existing \$50SAT designs and materials &  no more than 2 weeks \\
+    2 & Subsystem development & 6 weeks \\
+    2 & Order PCB design and/or assembly & 6 weeks \\
+    2 & Review changes to mechanical strucuture and order materials & no more than 2 weeks \\
+    2 & Testing of individual subsystems & 2 weeks \\
```

Comment:
* I think the phase column looks best if centered
* Instead of repeating 2, 2, 2, 2 ... I think it might look better and be more readable to put 2 at the first row and then no number for the remaining phase 2 items.
* I think the right column looks better when using "2 weeks or less" rather than "no more than 2 weeks" because it is consistent with the rest of the items having the number appear first
* I've been thinking about the issues with the table spanning 2 rows. I think we can force the table to be the width of one column using the `tabularx` package. I'm not always a fan of using more packages but this one might be pretty useful. more ideas here: https://tex.stackexchange.com/questions/82528/how-to-make-the-table-fit-into-the-column, https://tex.stackexchange.com/questions/312219/autofit-a-table-to-the-column-width

My suggested changes are included below
```tex
\usepackage{tabularx}

\begin{table}[htb]
  \caption{Estimated Timeline}
  \centering
  \begin{tabularx}{\columnwidth}{@{}cXl@{}} \toprule
    Phase & Task & Duration \\ \midrule
    1 & Review existing \$50SAT designs and materials &  2 weeks or less \\
    2 & Subsystem development & 6 weeks \\
      & Order PCB design and/or assembly & 6 weeks \\
      & Review changes to mechanical strucuture and order materials & 2 weeks or less \\
      & Testing of individual subsystems & 2 weeks \\
    3 & System Assembly & 1 week  \\
    4 & System testing & 2 weeks  \\
    5 & Generate documentation and delivery to SPEX & 1 week  \\
    \bottomrule
\end{tabularx}
\label{tab:timeline}
\end{table}
```